sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py

```diff
@@ -61,7 +61,7 @@ async def create_certificate(
 
         If this is the first version, the certificate resource is created. This operation requires the
         certificates/create permission. The poller requires the certificates/get permission, otherwise raises an
-        :class:`~azure.core.exceptions.HttpResponseError`
+        :class:`~azure.core.exceptions.HttpResponseError`.
```

Comment:
Still have :class:?

Reply:
It turns out that `next-pylint` is only upset about `:class:` markers in type annotations -- they're fine in docstring descriptions or `:returns:` and `:raises:` fields. See https://github.com/Azure/azure-sdk-for-python/pull/33448/#discussion_r1427488390